pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -24,7 +24,7 @@ workflow ImputationBeagle {
     String bed_suffix = ".bed"
     String bref3_suffix = ".bref3"
 
-    String gatk_docker = "broadinstitute/gatk:4.6.0.0"
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.6.0.0"
     String ubuntu_docker = "ubuntu:20.04"
```

Comment:
we might as well move off of all docker images onto GAR since the same issue could easily happen with a different image

Reply:
it looks like these tasks are ones that dont have GCR/GAR images

CalculateChromosomeLength
StoreChunksInfo
ErrorWithMessageIfErrorCountNotZero